script/Contracts.sol

```diff
@@ -26,8 +26,13 @@ library MainnetContracts {
     address public constant PROVIDER = address(123456789); // TODO: Update with deployed Provider
     address public constant BUFFER = address(987654321); // TODO: Update with deployed buffer
 
-    address public constant YNLSDE_WM = 0x8Face3283E20b19d98a7a132274B69C1304D60b4;
-    address public constant YNETH_WM = 0x0BC9BC81aD379810B36AD5cC95387112990AA67b;
+    address public constant WSTETH_WQ = 0x889edC2eDab5f40e902b864aD4d7AdE8E412F9B1;
+
+    address public constant YNLSDE_WQM = 0x8Face3283E20b19d98a7a132274B69C1304D60b4;
```

Comment:
lets make these names explicit  - i know they're long but i'd rather not think what they are